app/containers/SpeakEasies.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import CSSModules from "react-css-modules";
+import { connect } from "react-redux";
+import _ from "lodash";
+import styles from "../styles/containers/speak-easy.css";
+import { loadSpeakEasies, tapSpeakEasy } from "../actions/speakEasy";
+import SpeakEasyListItem from "../components/SpeakEasyListItem";
+
+class Location extends React.Component {
+  componentDidMount() {
+    const { dispatch } = this.props;
+
+    dispatch(loadSpeakEasies());
+  }
+
+  handleSpeakEasyTap(speakEasyId) {
+    const { dispatch } = this.props;
+
+    dispatch(tapSpeakEasy(speakEasyId));
+  }
+
+  renderSpeakEasies() {
+    const { speakEasies, activeId } = this.props;
+    return (
+      _.map(speakEasies, (speakEasy) => {
+        return <SpeakEasyListItem
+          key={speakEasy.id}
+          speakEasy={speakEasy}
+          activeId={activeId}
```

Comment:
Rather than making the `SpeakEasyListItem` have knowledge about the `activeId` I'd suggest just passing a prop for `isExpanded` and checking in this loop to see if it is or isn't active.

Try and make these other components as dumb as possible, makes them easier to re-use.